From 5d2968b8e7059f16ea3bd2d23114b13a9b9ad94d Mon Sep 17 00:00:00 2001
From: Montel Laurent <montel@kde.org>
Date: Thu, 7 Feb 2013 08:16:01 +0100
Subject: [PATCH 2/3] Don't try to start akonadiserver if mysqld is not install
 on distro (cherry picked from commit
 a2ad6ea98cfeae6a03ec02d9132d7c60dfbe9683)

---
 server/src/storage/dbconfigmysql.cpp | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/server/src/storage/dbconfigmysql.cpp b/server/src/storage/dbconfigmysql.cpp
index 6712338..c9491e0 100644
--- a/server/src/storage/dbconfigmysql.cpp
+++ b/server/src/storage/dbconfigmysql.cpp
@@ -262,6 +262,10 @@ void DbConfigMysql::startInternalServer()
   arguments << QString::fromLatin1( "--shared-memory" );
 #endif
 
+  if (mysqldPath.isEmpty()) {
+    akError() << "mysqld not found. Please verify your installation";
+    return;
+  }
   mDatabaseProcess = new QProcess;
   mDatabaseProcess->start( mysqldPath, arguments );
   if ( !mDatabaseProcess->waitForStarted() ) {
-- 
1.8.1.2

